### PR TITLE
Skip Active Directory tests

### PR DESCRIPTION
**Pull request by shuyama1**
The Active Directory tests never got passed in VCR or in our nightly runs. It takes quite a long time for the test to fail and therefore increases the VCR run time for every PR. I think we should skip the tests in VCR until we have them fixed. 
Plus, probably also skip the tests in nightly, since we are not able to fix them for now. 

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/tests/resource_active_directory_domain_update_test.go [hunk 1]
```diff
@@ -78,3 +83,36 @@ func testAccADDomainUpdate(context map[string]interface{}) string {
 	`, context)
 
 }
+
+func testAccCheckActiveDirectoryDomainDestroyProducer(t *testing.T) func(s *terraform.State) error {
```
Comment (c2thorn) [inline mmv1/third_party/terraform/tests/resource_active_directory_domain_update_test.go:87]: why is this needed?

**Review by c2thorn — APPROVED** [review]

File: mmv1/third_party/terraform/tests/resource_active_directory_domain_update_test.go (hunk 1 above)
Comment (shuyama1) [reply mmv1/third_party/terraform/tests/resource_active_directory_domain_update_test.go:87]: We used to have this function generated by mmv1 but the generated test file got removed since we skip all the tests. I just copied it from the previous generated file, otherwise the build is going to fail as we have the handwritten test `TestAccActiveDirectoryDomain_update` calling this function (even though it is also skipped)
